lib/generators/administrate/install/install_generator.rb

```diff
@@ -31,6 +37,13 @@ def run_dashboard_generators
         end
       end
 
+      def model_check
+        if valid_dashboard_models.none?
+          puts "** ERROR: Add models before installing Administrate. **"
```

Comment:
If this is not going to stop the generator, then I think that this should be a warning, not an error. Mind changing it to be similar to existing warnings? For example those at https://github.com/thoughtbot/administrate/blob/a5d695da773f15e1709e6be06c0a526156f6c1d6/lib/generators/administrate/routes/routes_generator.rb#L26